<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputBindingDialog</class>
 <widget class="QDialog" name="InputBindingDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Bindings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string>Bindings for Controller0/ButtonCircle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="bindingList"/>
   </item>
   <item>
    <widget class="QLabel" name="status">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addBinding">
       <property name="text">
        <string>Add Binding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBinding">
       <property name="text">
        <string>Remove Binding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBindings">
       <property name="text">
        <string>Clear Bindings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
